statsmodels/compat/_testing.py

```diff
@@ -0,0 +1,8 @@
+import platform
+
+__all__ = ['IS_OSX']
```

Comment:
Does this need it’s own module?

Reply:
It is now used in 2 distinct files, so I say yes.  It also prevents all of the `HAVE_MATPLOTLIB` style imports should this need arise again.